### DISCUSSION TITLE
但凡不是地图上夹带私活,我们还是愿意支持的

### DISCUSSION THREAD

**Discussion by TimothyHao** [WeNeedHome/SummaryOfLoanSuspension#652] [General]
有的的人反馈问题是为了趁机搞分裂,有的人反馈问题是希望问题得到解决
从地图来看,很明显是借机搞事的,既然是借机搞事,你觉得他们会不会让事情更乱?毕竟对于搞分裂的人,越乱越好.你在这去支持他们更加要不来房子
现在不管是B站还是微博,大家都在讨论也有热度,为什么你非要去一个搞分裂的人那里凑热度呢
是蠢吗?

**Comment by hktk1024** [comment]
拿着你的身份证去趟你所谓的“台湾省” 再来逼逼吧
不是你主子的就不是你主子的 是你主子的跟你也没一毛钱关系

**Comment by yooq** [comment]
你在这去支持他们更加要不来房子。----所以我不支持了对么？
谁搞分裂了，不团结在教主周围就是分裂吗？
不团结在人民周围的才是最大的分裂势力，滚出中国

**Comment by JACWATA** [comment]
有不同的声音就是搞分裂，跪太久起不来了吧

**Comment by MrChen2016-08-27** [comment]
把地图赶紧修改一下把，不然无法在国内平台活几秒，而且这图不是让递刀子让国内平台用户打烂尾楼业主？

**Reply by yangtuqiang** [reply]
的确，国内的话用geoserver，离线的比较好，可能就信息同步不太好

**Comment by yangtuqiang** [comment]
我看到说在B站和微博讨论我就笑了

**Comment by C-Joey** [comment]
曾母暗沙也有烂尾楼 一点都不能少

**Comment by yalo-ddaa** [comment]
建议你自己画一个，然后提交PR合并过来不就行了？有这份心就多做点实事。

**Comment by Yoobom** [comment]
哈哈，这就体现出源码的重要性了。建议楼主学习一下项目源码，应该在这里 https://github.com/WeNeedHome/SummaryOfLoanSuspension/blob/main/analysis/frontend/src/components
用的是google map，开源的，人家地图就这样，不是项目夹带私货。

**Comment by tencentbook** [comment]
请教下，地图上哪里夹带私活了

**Reply by Yoobom** [reply]
常见的开源组件涉及到的地图问题（边界，领土，争议地区等)，坑过国内很多公司

**Comment by hhx2018** [comment]
我还是倾向于认为Google地图源的问题，不要乱扣帽子，为这个项目做贡献的人也是为爱发电做出来了地图热力图，你如果不满意大可以使用你满意的开源地图来做一个替换。但是请不要随意给贡献者扣帽子

**Reply by MrChen2016-08-27** [reply]
确实是地图源得问题，刷新一下看到已经在撤了替换了
